### ISSUE TITLE
Unused work item tags not deleted within 3 days

### ISSUE THREAD

**Issue by felixtsem5** [MicrosoftDocs/azure-devops-docs#9784]
Under the [Delete, remove, or manage](https://docs.microsoft.com/en-us/azure/devops/boards/queries/add-tags-to-work-items?toc=%2Fazure%2Fdevops%2Fboards%2Ftoc.json&bc=%2Fazure%2Fdevops%2Fboards%2Fbreadcrumb%2Ftoc.json&view=azure-devops#delete-remove-or-manage-tags) section, it says that

> However, if you delete a tag from all work items to which it's currently assigned, the system will delete the tag. The system automatically deletes unassigned tags after 3 days of disuse.

I tried assigning a tag to work items and then removed the tags from all of those work items. After 3 days, I still see the tag available under tag suggestions when I add tags. Can you please clarify what the exact time frame that unused tags will be removed?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9bf3ce0a-184f-ca49-5453-01317e23dde5
* Version Independent ID: 0af9b10e-1126-2fda-982b-63d5795f9213
* Content: [Add tags to work items - Azure Boards](https://docs.microsoft.com/en-us/azure/devops/boards/queries/add-tags-to-work-items?toc=%2Fazure%2Fdevops%2Fboards%2Ftoc.json&bc=%2Fazure%2Fdevops%2Fboards%2Fbreadcrumb%2Ftoc.json&view=azure-devops#delete-remove-or-manage-tags)
* Content Source: [docs/boards/queries/add-tags-to-work-items.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/master/docs/boards/queries/add-tags-to-work-items.md)
* Product: **devops**
* Technology: **devops-agile**
* GitHub Login: @KathrynEE
* Microsoft Alias: **kaelli**

**Comment by KathrynEE** [comment]
@felixtsem5  - Thank you for your inquiry. I checked with the product manager and he indicated that the 3 days can vary. I recommend that you wait possibly 5 to 6 days, and if the tags haven't been removed by that time, raise an issue on our [Developer Community page](https://developercommunity.visualstudio.com/spaces/21/index.html).

**Comment by felixtsem5** [comment]
@KathrynEE - if the actual time is 5-6 days, can we please get the docs updated to reflect the actual time that it will take?

**Comment by KathrynEE** [comment]
@felixtsem5 - I'll make a note that it can vary. Thanks.

**Comment by felixtsem5** [comment]
On a Monday morning (6 days after I removed from the last work item), I found that the tag is indeed being removed from the system (can't find that in the tag list for querying). However, it is still being suggested when I try to add tags in the create/edit work item form. This can cause issues - mistyped tag is removed from the system but still suggested, and people can choose the mistyped tag again.
